app/ClientSrc/vue/coa/ApplicationTypeSelect.vue

```diff
@@ -0,0 +1,344 @@
+            @click="showModal"
+            type="button"
+            class="btn btn-radio btn-radio--secondary"
+        >
+            Choose Application Type(s)
+        </button>
+
+        <!-- modal dialog with selection options -->
+        <dialog ref="dialogEl">
+            <div class="title-wrap">
+                <h1 class="m-0">Choose application type(s)</h1>
+                <button class="btn-close" type="button" @click="$refs.dialogEl.close()">
+                    <i class="fas fa-times m-0"></i>
+                </button>
+            </div>
+
+            <ul class="options list-unstyled">
+                <li>
+                    <label class="option" v-for="option in formattedOptions" :key="option.id">
+                        <input type="checkbox" :value="option.id" v-model="newSelection" />
+
+                        <div class="label-text">
+                            <div class="font-weight-bold option-label">
+                                {{ option.label }}
+                            </div>
+
+                            <div class="font-weight-normal option-definition">
+                                {{ option.definition }}
+                            </div>
+                        </div>
+                    </label>
+                </li>
+            </ul>
+
+            <div class="actions-wrap">
+                <span>{{ newSelection.length }} selected</span>
+
+                <button class="btn btn-secondary mt-0" type="button" @click="confirmSelection">
+                    Confirm selection
+                </button>
+            </div>
+        </dialog>
+
+        <!-- expandable/removable panels to show the selection -->
+        <label class="application_type can-wrap"> Application type(s) selected: </label>
+
+        <div class="selection-display">
+            <div
+                class="selection-panel"
+                :class="{
+                    expandable: selected.definition.length,
+                    expanded: expandedPanels.includes(selected.id),
+                }"
+                v-for="selected in displaySelection"
+                :key="selected.id"
+                @click="toggleExpandPanel(selected)"
+            >
+                <div class="panel-label font-weight-bold">
+                    {{ selected.label }}
+                    <i
+                        v-if="selected.definition.length"
+                        class="ml-2 fas"
+                        :class="
+                            expandedPanels.includes(selected.id)
+                                ? 'fa-chevron-up'
+                                : 'fa-chevron-down'
+                        "
+                    ></i>
+                </div>
+
+                <div v-if="selected.definition.length" class="panel-definition">
+                    {{ selected.definition }}
+                </div>
+
+                <button class="btn-close" type="button" @click.stop="deselect(selected.id)">
```

Comment:
It doesn't look like these delete buttons are honouring the disable state of the control

Reply:
You're right! I disabled the modal dialog but forgot about the other part. I'll update that shortly 👍

Reply:
I made some tweaks to the button styles, and ended up just hiding these "deselect" X buttons when they're disabled because it looks a little cleaner than having a column of disabled buttons